hack/remove-calico-policy/calico-remove-iptables.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: calico-remove-iptables
+  namespace: kube-system
+  labels:
+    k8s-app: calico-remove-iptables
+spec:
```

Comment:
Could you add a simple `nodeSelector: {}` here which matches all nodes and a corresponding comment?  Will make it easier for folks to tweak in case they only want to run this on a select few nodes.